### PR TITLE
feat(Stability): update chain configuration and dependencies

### PR DESCRIPTION
**Pull request by deblanco**
## Summary

This PR adds support for [Stability GTN](https://github.com/stabilityprotocol/) network.

## Changes
- Chain configs

## Issues
NA

### CODE REVIEW

**Comment by deblanco** [comment]
I had to add the path for resolving `web-did-resolver` and `ethr-did-resolver` on `webpack.config.js` because they are not compatible with the same dependencies on the `@tradetrust-tt/tt-verify`.
Consider bumping the Webpack version to 5 to avoid these problems and help with the dev experience (it took a while)

image

**Comment by deblanco** [comment]
@isaackps, it seems that there is a problem with the `ethers` version. Several packages updated it to v5.7.2, where the BaseProvider and Contract type imports are moved to the main package instead of the `@etherproject` package as they used to be.

Also, the `ether-did-resolver` and `web-did-resolver` can be found in different versions that break the API, having to fix the version.

**Comment by isaackps** [comment]
yeah, understand the differences in the packages, Im currently working together with my team to fix this throughout all the repos. Thanks for highlighting it. most probably, we will revert the changes to the packages in those repo.

**Comment by isaackps** [comment]
I have just merged a fix for the dependencies, we revert the changes back to how it was previously, we should not need to force resolution and the dependencies should not be an issue anymore. 

Please rebase the latest master and in this PR it should only contain stability network details. 

Thanks, and sorry about the messy dependency issue.

**Comment by deblanco** [comment]
> I have just merged a fix for the dependencies, we revert the changes back to how it was previously, we should not need to force resolution and the dependencies should not be an issue anymore. 
> 
> 
> 
> Please rebase the latest master and in this PR it should only contain stability network details. 
> 
> 
> 
> Thanks, and sorry about the messy dependency issue.

Seems we are ready now

File: src/config/index.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ module.exports = {
     allow_ad_personalization_signals: false,
     debug_mode: IS_DEVELOPMENT,
   },
+  STABILITY_API_KEY: process.env.STABILITY_API_KEY || "test",
```
Comment (isaackps) [inline src/config/index.js]: does "test" api key work? if not we can install dotenv and use tradetrust api key from there.

Comment (deblanco) [reply src/config/index.js]: hey it doesn't, let me remove it

**Comment by deblanco** [comment]
@isaackps ready

**Review by isaackps — APPROVED** [review]

**Pull request merged**